web_src/js/features/notification.js

```diff
@@ -0,0 +1,67 @@
+const {AppSubUrl, csrf} = window.config;
+
+export async function initNotificationsTable() {
+  $('#notification_table .button').click(function () {
+    updateNotification(
+      $(this).data('url'),
+      $(this).data('status'),
+      $(this).data('page'),
+      $(this).data('q'),
+      $(this).data('notification-id')
+    ).then((data) => {
+      $('#notification_div').replaceWith(data);
+      initNotificationsTable();
+      updateNotificationCount();
+    });
+    return false;
+  });
+}
+
+export async function initNotificationCount() {
```

Comment:
dont init this if user is not loged in !!

I'm geting a lot of 401 errors in my test instance

Reply:
Should be fixed now.